journals/apps/api/v1/views.py

```diff
@@ -25,6 +24,11 @@ def create(self, request, *args, **kwargs):
         '''create a JournalAccess entry'''
         username = request.data.get('user')
         journal_uuid = request.data.get('journal')
+        order_number = request.data.get('order_number', None)
+        revoke_access = request.data.get('revoke_access', 'false').lower() == 'true'
+        if revoke_access:
+            return self.revoke_access(order_number)
```

Comment:
NIT: should probably rename revoke_access to _revoke_access() as it's an internal method and then call self._revoke_access()